.env.sample

```diff
@@ -1,15 +1,30 @@
+# Database connection string
 DATABASE_URL=postgresql://postgres@localhost:5432/tefca_db
+
+# Authentication settings
+AUTH_DISABLED=false
+NEXT_PUBLIC_AUTH_PROVIDER=keycloak
 AUTH_SECRET="ido5D/uybeAB3AmMQwn+ubw2zYC4t2h7RJlW2R79598="
+
+# Keycloak settings - set NEXT_PUBLIC_AUTH_PROVIDER to "keycloak" to use these settings
 AUTH_KEYCLOAK_ID=query-connector
 AUTH_KEYCLOAK_SECRET=ZG3f7R1J3qIwBaw8QtttJnJMinpERQKs
 AUTH_KEYCLOAK_ISSUER=http://localhost:8081/realms/master
-AUTH_DISABLED=false
-DEMO_MODE=true
-# Need both of these so docker-compose.yaml can overwrite them when communicating with auth.ts
 LOCAL_KEYCLOAK=http://localhost:8081
 NAMED_KEYCLOAK=http://localhost:8081
+
+# Entra ID settings - set NEXT_PUBLIC_AUTH_PROVIDER to "microsoft-entra-id" to use these settings
+AUTH_MICROSOFT_ENTRA_ID_ID=your-client-id
+AUTH_MICROSOFT_ENTRA_ID_SECRET=your-client-secret
+AUTH_MICROSOFT_ENTRA_ID_ISSUER=https://login.microsoftonline.com/your-tenant-id/v2.0
```

Comment:
do we think we'll need to add similar env vars as we add additional IDP's? this can come in a followup, but wonder if we could just ask for the set of ID/secret/issuer and then look at the AUTH_PROVIDER to extrapolate which AuthJS interface we'd need to feed those vars into

Reply:
Yeah, given the way we are reading the values explicitly in `auth.ts` anyways instead of relying on these var names that get picked up implicitly, using just a single set of vars is doable and preferable! Good callout!

Reply:
Done!